k2/csrc/fsa_algo.cc

```diff
@@ -374,8 +362,8 @@ bool Intersect(const Fsa &a, const Fsa &b, Fsa *c,
       auto b_arc_range =
           std::equal_range(b_arc_iter_begin, b_arc_iter_end, curr_a_arc,
                            [](const Arc &left, const Arc &right) {
-                             return left.label < right.label;
-                           });
+            return left.label < right.label;
```

Comment:
what is your clang-format version ?

I am using 9.0.0.

Reply:
same version...